docs/tutorials/send_receive_ingress_pem.md

```diff
@@ -51,24 +64,51 @@ If you are running your OpenShift cluster locally, you might not be able to
 resolve the urls to IPs out of the blue. Follow [this guide]({{< ref
 "/docs/help/hostname_resolution" >}} "set up dnsmasq") to configure your setup.
 
+This tutorial will follow the simple /etc/hosts approach, but feel free to use
+the most appropriate one for you.
+
 ### Deploy the operator
 
 #### create the namespace
 
-```console
-$ kubectl create namespace send-receive-project
-$ kubectl config set-context --current --namespace=send-receive-project
+```{"stage":"init"}
+kubectl create namespace send-receive-project
+kubectl config set-context --current --namespace=send-receive-project
+```
+```shell tutorial_tester
+namespace/send-receive-project created
+Context "tutorialtester" modified.
 ```
 
 Go to the root of the operator repo and install it:
 
-```console
-$ ./deploy/install_opr.sh
+```{"stage":"init", "rootdir":"$operator"}
+./deploy/install_opr.sh
+```
+```shell tutorial_tester
+Deploying operator to watch single namespace
+Client Version: 4.15.0-0.okd-2024-01-27-070424
+Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
+Kubernetes Version: v1.28.3
+customresourcedefinition.apiextensions.k8s.io/activemqartemises.broker.amq.io created
+customresourcedefinition.apiextensions.k8s.io/activemqartemisaddresses.broker.amq.io created
+customresourcedefinition.apiextensions.k8s.io/activemqartemisscaledowns.broker.amq.io created
+customresourcedefinition.apiextensions.k8s.io/activemqartemissecurities.broker.amq.io created
+serviceaccount/activemq-artemis-controller-manager created
+role.rbac.authorization.k8s.io/activemq-artemis-operator-role created
+rolebinding.rbac.authorization.k8s.io/activemq-artemis-operator-rolebinding created
+role.rbac.authorization.k8s.io/activemq-artemis-leader-election-role created
+rolebinding.rbac.authorization.k8s.io/activemq-artemis-leader-election-rolebinding created
+deployment.apps/activemq-artemis-controller-manager created
 ```
 
 Wait for the Operator to start (status: `running`).
-```console
-$ kubectl get pod --namespace send-receive-project --watch
+
+```{"stage":"init"}
+kubectl wait pod --all --for=condition=Ready --namespace=send-receive-project --timeout=240s
```

Comment:
The provided tls secret is created after 2~3 minutes and on a laptop with a slow internet connection the download of the container images could require other 2~3 minutes.
```suggestion
kubectl wait pod --all --for=condition=Ready --namespace=send-receive-project --timeout=600s
```